test/integration/infrastructure/infrastructure_test.go

```diff
@@ -147,6 +145,9 @@ var _ = Describe("Infrastructure tests", func() {
 
 		restConfig = k8sClient.RESTConfig()
 		c = k8sClient.Client()
+
+		chartRenderer, err = chartrenderer.NewForConfig(restConfig)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
This can be simplified now.
```suggestion
		chartRenderer = k8sClient.ChartRenderer()
```